### PR TITLE
Add 2023.12 release notes to .metainfo.xml

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

File: packaging/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -113,6 +113,28 @@
   </screenshots>
 
   <releases>
+   <release version="2023.12.18-11" date="2023-12-18">
```
Comment (biodranik) [inline packaging/app.organicmaps.desktop.metainfo.xml:116]: Let's wait for the follow-up hotfix release.

Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml]: Done

Comment (biodranik) [reply packaging/app.organicmaps.desktop.metainfo.xml]: • Outdoors map style for hiking, cycling, and exploring the Nature. Type ?olight or ?odark in the search to enable it, and ?light or ?dark to disable it. Any volunteers to add convenient buttons are welcome!
• Search for village addresses without streets in Austria, Czechia, Germany, Poland, Slovakia and some other countries (addr:place OSM tag support)
• Type "skiing" or "ski run" to find downhill and Nordic pistes
• OpenStreetMap map data as of December 13
• Multiple KML files can be imported from a single KMZ file
• A credit card 💳 is displayed for selected places with ATM
• A dollar emoji 💲 is displayed where a fee is required

Styles
• Please provide feedback on how to improve the Outdoors style
• Singapore metro icon
• Added Hackerspace
• Fixed Porto metro icon
• Fixed residential leisure garden area fills disappearing on some zoom levels
• Display house numbers regardless of buildings' sizes on zoom level 17
• Fine-tuned priorities of many POIs in the main style
• Less "gaps" in highways on the World map
• Removed very short ferry lines from the World map

Translations
• Improved Chinese, Dutch, Finnish, Polish, Russian translations
• Added Dutch, French, Polish, Romanian, and Russian translations for edition Opening Hours instructions
• Fixed missing parking translations

Linux
• Show Wikipedia articles in a separate dialog
• Touchscreen pinch-zoom support

Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml]: Thank you. Added.

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**